src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/claimsbyhearingvenue/ClaimsByHearingVenueReportParams.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.reports.claimsbyhearingvenue;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import uk.gov.hmcts.ethos.replacement.docmosis.reports.ReportParams;
+
+@Value
```

Comment:
Personally I don't like this. I don't see the value of this over explicitly putting private final on your variableas

Reply:
Ok. I can make it simpler and explicit with @Getter and @Setter.